src/libtiledquick/libtiledquick.qbs

```diff
@@ -0,0 +1,72 @@
+import qbs 1.0
+
+DynamicLibrary {
+    targetName: "tiledquick"
+
+    Depends { name: "libtiled" }
+    Depends { name: "cpp" }
+    Depends { name: "Qt"; submodules: ["gui", "quick"]; versionAtLeast: "5.6" }
+
+    cpp.cxxLanguageVersion: "c++14"
+    cpp.visibility: "minimal"
+    cpp.defines: [
+        "TILED_QUICK_LIBRARY",
+        "QT_NO_CAST_FROM_ASCII",
+        "QT_NO_CAST_TO_ASCII",
+        "QT_NO_URL_CAST_FROM_STRING",
+        "_USE_MATH_DEFINES",
```

Comment:
Should also include `QT_DEPRECATED_WARNINGS`, `QT_DISABLE_DEPRECATED_BEFORE=0x050900` and `QT_NO_FOREACH`, or did you have some reason to leave those out?

Reply:
I copied directly from libtiled. I can add them if you want?

Reply:
Right, but the code is copied from `tiledquickplugin`, so please copy the defines from there. :-)

You should remove `_USE_MATH_DEFINES` as well, since there's probably no need for it here.